docs/config.md

```diff
@@ -436,3 +436,17 @@ spec:
         value: ok
         effect: NoExecute
 ```
+
+### Target Discovery Options
+
+If you wish to use only Cryostat's Discovery Plugin API, the property `spec.targetDiscoveryOptions.builtInDiscoveryDisabled` to `true` to disable Cryostat's built-in discovery mechanisms.
```

Comment:
Probably worth linking to this doc: https://github.com/cryostatio/cryostat/blob/main/docs/DISCOVERY_PLUGINS.md